tests/finish-branch-zsh.test.sh

```
@@ -406,6 +411,379 @@ else
+  [ -f "$FBT1_ROOT/tests-ran" ] || fbt_bad 1-tests-never-ran
+  fbt_registered "$FBT1_ROOT" "$FBT1_ROOT/.worktrees/feat-x" && fbt_bad 1-worktree-still-registered
+  [ -d "$FBT1_ROOT/.worktrees/feat-x" ] && fbt_bad 1-worktree-dir-survived
+  fbt_has_branch "$FBT1_ROOT" feat/x && fbt_bad 1-branch-not-deleted
+  if [ -z "$FBT_FAILURES" ]; then
+    pass "FBT-1: from a linked worktree the merge actually LANDS on the base, the worktree is removed and the branch deleted (the old prose reported this merge without performing it)"
+  else
+    fail "FBT-1: linked-worktree merge path broken:$FBT_FAILURES"
+    printf '        stdout=[%s]\n' "$(printf '%s' "$FBT_OUT" | tr '\n' '|')"
+    printf '        stderr=[%s]\n' "$(printf '%s' "$FBT_ERR" | tr '\n' '|')"
+  fi
+  FBT1_FAILURES="$FBT_FAILURES"; FBT_FAILURES=''
```

Shellcheck (0.11.0)

[warning] FBT1_FAILURES appears unused. Verify use (or export if used externally).

(SC2034)